addon/components/cfb-form-editor/question.js

```diff
@@ -420,6 +429,17 @@ export default Component.extend(ComponentQueryManager, {
       this.set("linkSlug", false);
 
       this.get("validateSlug").perform(this.prefix + value, changeset);
+    },
+
+    selectColumnToDisplay(value) {
+      let arr = this.get("model.selectedColumnsToDisplay");
+      let idx = arr.indexOf(value);
+      if (idx >= 0) {
+        arr.splice(idx, 1);
+        this.set("model.selectedColumnsToDisplay", arr);
+      } else {
+        this.set("model.selectedColumnsToDisplay", [...arr, value]);
+      }
```

Comment:
Took me a moment to figure out what you're doing here - maybe `toggleColumnToDisplay` would be a better name? And/or add a comment explaining that if the function receives a slug that is already select, it will deselect it?

Reply:
Renamed and added a comment